com.unity.perception/Documentation~/Randomization/Parameters.md

```diff
@@ -24,6 +24,31 @@ All parameters derive from the `Parameter` abstract class, but all included perc
 1. `CategoricalParameter`
 2. `NumericParameter`
 
+## Using Parameters outside of Randomizers (ie: in MonoBehaviours and ScriptableObjects)
+
+After adding a public Parameter field to a Monobehavior or ScriptableObject, you may have noticed that the parameter's UI doesn't look the same as it does on when added to a Randomizer. This is because the Inspector UI for most Perception randomization components are authored using Unity's relatively new UI Elements framework, though by default, Unity uses the old IMGUI framework to render default inspector editors.
```

Comment:
```suggestion
After adding a public Parameter field to a MonoBehaviour or ScriptableObject, you may have noticed that the Parameter's UI doesn't look the same as it does when added to a Randomizer. This is because the Inspector UI for most Perception randomization components is authored using Unity's relatively new UI Elements framework, though by default, Unity uses the old IMGUI framework to render default inspector editors.
```